src/components/Common/RelativeDateTooltip.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTranslation } from "react-i18next";
+
+import { TooltipComponent } from "@/components/ui/tooltip";
+
+import dayjs from "@/Utils/dayjs";
+import { formatDateTime } from "@/Utils/utils";
+
+type RelativeDateTooltipProps = {
+  date: string | number | Date | dayjs.Dayjs | null | undefined;
```

Comment:
why allow undefined or null? 🤔 



```suggestion
  date: string | number | Date | dayjs.Dayjs | null | undefined;
```